app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,6 +89,7 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      self.class.make_ems_ref(import.vm.href)
```

Comment:
I know that the fully qualified name of this method is very long, but I'd say that assuming that it is available in this class isn't correct. Besides, if this code is moved to a different place, it will not work. I think it would be better to use the ugly fully qualified name, but that is probably a matter of taste. So +0.99 from me.